Gems/Terrain/Assets/Shaders/Terrain/TerrainPBR_ForwardPass.azsl

```diff
@@ -30,6 +30,7 @@ struct VSOutput
     float3 m_normal: NORMAL;
     float3 m_worldPosition : UV0;
     float3 m_shadowCoords[ViewSrg::MaxCascadeCount] : UV2;
+    float2 m_cullDistance : SV_CullDistance;
```

Comment:
Will do (it's because we clip on both a vertex being too close or too far away from the camera for its LOD range). We should be able to reduce this to just one clipping plane when continuous LOD is implemented, but doing it on both sides provides a little bit of overlap between lods which helps reduce seams.